sympy/printing/latex.py

```diff
@@ -39,35 +39,35 @@
                             ]
 
 tex_greek_dictionary = {
-    'Alpha': 'A',
-    'Beta': 'B',
+    'Alpha': r'\mathrm{A}',
+    'Beta': r'\mathrm{B}',
     'Gamma': r'\Gamma',
     'Delta': r'\Delta',
-    'Epsilon': 'E',
-    'Zeta': 'Z',
-    'Eta': 'H',
+    'Epsilon': r'\mathrm{E}',
+    'Zeta': r'\mathrm{Z}',
+    'Eta': r'\mathrm{H}',
     'Theta': r'\Theta',
-    'Iota': 'I',
-    'Kappa': 'K',
+    'Iota': r'\mathrm{I}',
+    'Kappa': r'\mathrm{K}',
     'Lambda': r'\Lambda',
-    'Mu': 'M',
-    'Nu': 'N',
+    'Mu': r'\mathrm{M}',
+    'Nu': r'\mathrm{N}',
     'Xi': r'\Xi',
     'omicron': 'o',
```

Comment:
```suggestion
    'omicron': r'\mathrm{o}',
```
for consistency?

(I note that TeXStudio uses the default for the their Greek list, so no `\mathrm`, but I understand why that may be incorrect.)

Reply:
I think it is just the capitals that are supposed to be upright. See e.g. [this answer](https://tex.stackexchange.com/a/119262/200616).

Reply:
Curiously, the `O` in big O notation is supposedly an omicron, but I'm not sure I've ever seen it presented as upright.